### ISSUE TITLE
Title

### ISSUE THREAD

**Issue by malliaanthony** [rose2hnd1i/IMTAssignment2016#5]
While playing the game I have notice that the game has no title that represents it. Will you be giving it name? What would it be?

**Comment by rose2hnd1i** [comment]
Yes, of course. the name of the game is going to be 'turtle life' since it focuses on sea turtles and how they are affected by water pollution.
